R/model.R

```diff
@@ -1080,364 +243,46 @@ late_fall_run_model <- function(scenario = NULL, mode = c("seed", "simulate", "c
-                                     .pulse_movement_very_large_pulse = ..params$.pulse_movement_very_large_pulse,
-                                     tempDwnStrm=params$avg_temp_delta[ month, year,1], #north delta
-                                     territory_size = lateFallRunDSM::params$territory_size)
-        
-        lower_sac_trib_rear <- rear(juveniles = lower_sac_trib_fish$inchannel,
-                                    survival_rate = rearing_survival$inchannel[23, , drop = FALSE],
-                                    growth = lateFallRunDSM::params$growth_rates,
-                                    floodplain_juveniles = lower_sac_trib_fish$floodplain,
-                                    floodplain_survival_rate = rearing_survival$floodplain[23, , drop = FALSE],
-                                    floodplain_growth = lateFallRunDSM::params$growth_rates_floodplain,
-                                    weeks_flooded = ..params$weeks_flooded[23, month, year])
-        
-        juveniles3[23, ] <- lower_sac_trib_rear$inchannel + lower_sac_trib_rear$floodplain
-        
-        migrants[23, ] <- lower_sac_trib_fish$migrants
-        
-        lower_sac_fish <- route_regional2(month = month,
-                                         migrants = lower_sac_fish + migrants,
-                                         inchannel_habitat = habitat$inchannel[24],
-                                         floodplain_habitat = habitat$floodplain[24],
-                                         prop_pulse_flows = ..params$prop_pulse_flows[24, , drop = FALSE],
-                                         migration_survival_rate = migratory_survival$lower_sac,
-                                         tempDwnStrm=params$avg_temp_delta[ month, year,1], #north delta
-                                         territory_size = lateFallRunDSM::params$territory_size)
-        
-        migrants <- lower_sac_fish$migrants
-        
-        lower_sac_fish <- rear(juveniles = lower_sac_fish$inchannel,
-                               survival_rate = rearing_survival$inchannel[24, ],
-                               growth = lateFallRunDSM::params$growth_rates,
-                               floodplain_juveniles = lower_sac_fish$floodplain,
-                               floodplain_survival_rate = rearing_survival$floodplain[24, ],
-                               floodplain_growth = lateFallRunDSM::params$growth_rates_floodplain,
-                               weeks_flooded = rep(..params$weeks_flooded[24, month, year], nrow(lower_sac_fish$inchannel)))
-        
-        lower_sac_fish <- lower_sac_fish$inchannel + lower_sac_fish$floodplain
-        
-        # route southern natal fish stay and rear or migrate downstream ------
-        
-        # route migrant fish into South Delta Region (fish from watersheds 25:27)
-        # regional fish stay and rear
-        # or migrate to south delta
-        south_delta_trib_fish <- route2(year = year,
```

Comment:
dump into north delta, so remove temperature barrier for this...or hardcode in that it is <18C